README.md

```diff
@@ -297,18 +296,35 @@ times for multiple relays. Use whichever method suits your preferences.
 
 These two MEV-Boost commands are equivalent:
 
-```
-./mev-boost -mainnet -relay-check \
+```bash
+./mev-boost -relay-check \
     -relays $YOUR_RELAY_CHOICE_A,$YOUR_RELAY_CHOICE_B,$YOUR_RELAY_CHOICE_C
 ```
 
+```bash
+./mev-boost -relay-check \
+    -relay $YOUR_RELAY_CHOICE_A \
+    -relay $YOUR_RELAY_CHOICE_B \
+    -relay $YOUR_RELAY_CHOICE_C
 ```
-./mev-boost -mainnet -relay-check \
-    -relay $YOUR_RELAY_CHOICE_A   \
-    -relay $YOUR_RELAY_CHOICE_B   \
+
+
+### Setting a minimum bid value with `-min-bid`
+
+The `-min-bid` flag allows setting a minimum bid value. If no bid from the builder network delivers at least this value, mev-boost will not return a bid
```

Comment:
For consistency, replace "mev-boost" with "MEV-Boost".

Reply:
https://github.com/flashbots/mev-boost/pull/420